CHANGES.rst

```diff
@@ -49,6 +49,8 @@ Specviz
 Other Changes and Additions
 ---------------------------
 
+- Change default collapse function to sum. [#1229]
+
```

Comment:
Is this cubeviz-specific (does it belong under the cubeviz header)?

Reply:
It's not an API change, bug fix, or new feature as far as I could tell, so I put it in the "other" category. Since the change happens in the Specviz viewer file but is most prominent in Cubeviz, I'm not entirely sure where it goes!